setup/backup.class.inc.php

```diff
@@ -466,11 +466,10 @@ protected function EnumerateTables()
 	 * @return string TLS arguments for CLI programs such as mysqldump. Empty string if the config does not use TLS.
 	 *
 	 * @uses \CMDBSource::GetDBVendor() so needs a connection opened !
-	 * @uses \CMDBSource::GetDBVersion() so needs a connection opened !
 	 *
 	 * @since 2.5.0 N°1260
 	 * @since 2.6.2 2.7.0 N°2336 Call DB to get vendor and version (so CMDBSource must be init before calling this method)
-	 * @link https://dev.mysql.com/doc/refman/5.6/en/connection-options.html#encrypted-connection-options "Command Options for Encrypted Connections"
+	 * @link https://dev.mysql.com/doc/refman/5.7/en/connection-options.html#encrypted-connection-options "Command Options for Encrypted Connections"
```

Comment:
Thanks for the change !
You can remove the quotes around the title, see https://docs.phpdoc.org/guide/references/phpdoc/tags/link.html#link

```suggestion
	 * @link https://dev.mysql.com/doc/refman/5.7/en/connection-options.html#encrypted-connection-options Command Options for Encrypted Connections
```

I didn't get what you were saying in your comment ? 

> I thought it would be better to set the anchor to the top chapter (encrypted-connection-options) since the depreciation info of --ssl is not the the chapter you mentioned.

The old and the new URL are pointing to the same `encrypted-connection-options` anchor ? And in the 5.6 URL there is the `--ssl` parameter documented ?

Reply:
> You can remove the quotes around the title
Will do that 👍 

> I didn't get what you were saying in your comment ?

OK now I am not sure whether I got what you are saying 😀 
In your comment at the top you suggested to change the link to 5.7 (instead of 5.6). But you suggested a link to the sub-chapter for the `--ssl-mode`-option 
<img width="756" alt="image" src="https://user-images.githubusercontent.com/24569763/200405550-2cf72713-dd12-407a-8148-e96146f30d49.png">

Since the option `--ssl-mode` would be only used for mysql >= 5.7.11 and vor older versions `--ssl` IMHO it would make more sense to link to the top chapter (encrypted-connection-options) which includes the description von the sub chapter for `--ssl` and `--ssl-mode`

Does my explanation make sense to you or do you prefer the link to the `--ssl-mode`-sub chapter?

Reply:
Oh yeah sorry :D

Totally agree to link to https://dev.mysql.com/doc/refman/5.7/en/connection-options.html#encrypted-connection-options ("Command Options for Encrypted Connections" chapter)  !
Instead of https://dev.mysql.com/doc/refman/5.7/en/connection-options.html#option_general_ssl-mode (--ssl_mode param) I suggested earlier, which is indeed a bad idea !

Sorry for the confusion, thanks for the clarification :)

Reply:
Problem solved, closing conversation